src/test/scala/zio/sqs/producer/ProducerSpec.scala

```diff
@@ -198,23 +198,25 @@ object ProducerSpec extends DefaultRunnableSpec {
                           .run(Sink.head[Chunk[ProducerEvent[String]]])
                           .someOrFailException
           retryQueue <- queueResource(16)
-          dones      <- serverResource.useDiscard {
-                          retryQueue.use {
-                            q =>
-                              for {
-                                _          <- Utils.createQueue(queueName)
-                                queueUrl   <- Utils.getQueueUrl(queueName)
-                                reqEntries <- ZIO.foreach(events) { event =>
-                                                for {
-                                                  done <- Promise.make[Throwable, ErrorOrEvent[String]]
-                                                } yield SqsRequestEntry[String](event, done, 0)
-                                              }
-                                req         = Producer.buildSendMessageBatchRequest[String](queueUrl, Serializer.serializeString)(reqEntries.toList)
-                                retryDelay  = 1.millisecond
-                                retryCount  = 1
-                                reqSender   = Producer.runSendMessageBatchRequest(q, retryDelay, retryCount) _
-                                _          <- reqSender(req)
-                              } yield ZIO.foreach(reqEntries)(entry => entry.done.await)
+          dones      <- ZIO.scoped {
```

Comment:
I'm not sure whether this is the right place for this, but it fails both here and outside the `for`.